lib/Conversion/TorchOnnxToTorch/DefaultDomainAtoF.cpp

```diff
@@ -35,6 +35,116 @@ static LogicalResult createTorchTransposeOp(ConversionPatternRewriter &rewriter,
   return success();
 }
 
+namespace {
+LogicalResult windowFunctionImpl(OpBinder binder,
+                                 ConversionPatternRewriter &rewriter,
+                                 Value size, Value a0, Value a1, Value a2,
+                                 Torch::ValueTensorType resultType,
+                                 int64_t output_datatype, int64_t periodic) {
+
+  double isPeriodicFp = static_cast<double>(periodic);
+
+  Value zero = rewriter.create<Torch::ConstantFloatOp>(
```

Comment:
consider either using a Loc variable to replace all the binder.getLoc

or use an `mlir::ImplicitLocOpBuilder b(loc, rewriter);` to reduce code complexity & line length. example here: https://github.com/llvm/torch-mlir/pull/2969/files#diff-a89c31dc3f7349d89fe1d51c41ab28d81e2b88530ae79007bb1958e21d864d32R252

in lib/Conversion/TorchOnnxToTorch/OnnxLstmExpander.cpp:249

Reply:
Fixed in the latest commit, the code does look much cleaner now. Thanks for this tip!